solver/pb/ops.pb.go

```diff
@@ -929,7 +1041,7 @@ func (m *BuildOp) Reset()         { *m = BuildOp{} }
 func (m *BuildOp) String() string { return proto.CompactTextString(m) }
 func (*BuildOp) ProtoMessage()    {}
 func (*BuildOp) Descriptor() ([]byte, []int) {
-	return fileDescriptor_8de16154b2733812, []int{10}
+	return fileDescriptor_8de16154b2733812, []int{12}
```

Comment:
Silly question (I'm horrible at protobufs); I see these values changed for various descriptors; should these be "fixed" (as in: don't change)? Not sure what the impact is on changing these?

Reply:
Any changes made in `.proto` files are expected to cause the file descriptors to change. It's an internal implementation detail of the generated code so don't think it's documented somewhere. No need to worry about what's changing in the generated code as long as the protos follow the documented backwards-compatibility rules.